ci/verify-chef.sh

```diff
@@ -8,6 +8,9 @@ export TMPDIR
 sudo rm -rf $TMPDIR
 mkdir -p $TMPDIR
 
+# Verify that we kill any orphaned test processes. Kill any orphaned rspec processes.
+ps ax | egrep 'rspec' | grep -v grep | awk '{ print $1 }' | xargs sudo kill -s TERM || true
```

Comment:
`ps ax` make solaris or aix angry?

also you should just SIGKILL and nuke from orbit.  it's the only way to be sure...

Reply:
Done

Reply:
Also `ps ax` works on sol and aix, the flags with `-` sometime cause issues.